virttest/virsh.py

```diff
@@ -2908,14 +2908,17 @@ def snapshot_info(name, snapshot, **dargs):
     # CmdResult is handled here, force ignore_status
     dargs['ignore_status'] = True
     ret = {}
-    values = ["Name", "Domain", "Current", "State", "Parent",
+    values = ["Domain", "Current", "State", "Parent",
               "Children", "Descendants", "Metadata"]
 
     cmd = "snapshot-info %s %s" % (name, snapshot)
     sc_output = command(cmd, **dargs)
     if sc_output.exit_status != 0:
         raise process.CmdError(cmd, sc_output, "Failed to get snapshot info")
 
+    data = re.search("(?<=Name:) *[\w-]*", sc_output.stdout)
```

Comment:
try use regexp:"(?<=Name:) *\w.*"  ,which make it compatable with avocado-vt-vm1

Reply:
@chunfuwen Done